addons/payment_stripe/models/payment_transaction.py

```diff
@@ -190,7 +190,7 @@ def _stripe_create_customer(self):
                 'address[postal_code]': self.partner_zip or None,
                 'address[state]': self.partner_state_id.name or None,
                 'description': f'Odoo Partner: {self.partner_id.name} (id: {self.partner_id.id})',
-                'email': self.partner_email or None,
+                'email': self.partner_email,
```

Comment:
I don't think that you can remove that. You set `None` in `create` but, when you read the value, the ORM casts `null` to `False`.

Reply:
I confirmed. Well spotted.